### ISSUE TITLE
Table overflowing

### ISSUE THREAD

**Issue by ghaith-arabkerly** [pdfme/pdfme#580]
### Describe the bug

Table overflowing on a text when adding a new row

### To Reproduce

add a table and a text beneath and start adding rows

### Expected behavior

rows push the text under it

### Your Environment

```markdown
- pdfme package(@pdfme/generator or @pdfme/ui):
- pdfme version:
- Operating system:
- Node.js version or Browser name & version:
```

### Your Error Log

```shell
---
```

### Additional context

https://github.com/user-attachments/assets/f88812e7-9690-4ef5-917f-5d730fe67c3a

**Comment by ghaith-arabkerly** [comment]
@hand-dot please I need to use it , it's really urgent I tried to solve it but I couldn't

**Comment by Meadril** [comment]
@ghaith-arabkerly This function is not currently implemented, as far as I can see. I haven't looked through the complete code yet.
But what I can say is that it is the same with all components and to be honest I think the performance would suffer greatly if all components had to be repositioned as soon as components meet.

**Comment by ghaith-arabkerly** [comment]
@Meadril I don't think so , everything works fine on the first page , but then a table is added in the second page it doesn't push the elements beneath it , so what's the difference if the overflowing mechanism is applied to all pages ?

**Comment by Meadril** [comment]
@ghaith-arabkerly That's strange, it doesn't even work for me on the first page and I can't find anything in the code.
